Userland/Services/KeyboardPreferenceLoader/main.cpp

```diff
@@ -45,6 +45,11 @@ int main()
     auto mapper_config(Core::ConfigFile::open("/etc/Keyboard.ini"));
     auto keymap = mapper_config->read_entry("Mapping", "Keymap", "");
 
+    //Check the keymap changed using the GUI tool
```

Comment:
This comment doesn't seem to be necessary.

Reply:
I'm happy to yeet it